source/includes/fundamentals/code-snippets/Geo.java

```diff
@@ -63,13 +70,20 @@ private void rangeExample() {
         MongoCollection<Document> collection = database.getCollection("theaters");
         // begin rangeExample
 
-        // code to set up your mongo collection ...
+        // Add your MongoCollection setup code here
```

Comment:
Suggestion:
I think it would be good to include a linebreak between two separate comments so it can be read and understood as a separate comment. I think the blank line after the Marker comment ("begin rangeExample") is not necessary.